datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -648,11 +649,32 @@ impl AggregateExec {
         group_expr_mapping: &ProjectionMapping,
         mode: &AggregateMode,
         input_order_mode: &InputOrderMode,
+        aggr_exprs: Vec<Arc<AggregateFunctionExpr>>,
```

Comment:
A cursory look suggests this can be a slice. If that is true, let's avoid copying.